ceno_zkvm/src/structs.rs

```diff
@@ -254,6 +254,10 @@ impl<E: ExtensionField> ZKVMWitnesses<E> {
         self.witnesses_tables.get(name).cloned()
     }
 
+    pub fn get_lk_mlt(&self, name: &String) -> Option<LkMultiplicity> {
```

Comment:
we should move `deep_clone()` logic to mock_prover and here should just return Option<&LkMultiplicity> in case of being abuse in production code in the future, since deep_clone is very heavy